File: src/Chapter05/SlickUpload-5.5.1/AdvancedSamples/AspNetMvcCS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Web;
using System.Web.Mvc;
using Krystalware.SlickUpload.Controls;
using Krystalware.SlickUpload.Status;

namespace AspNetMvcCS.Controllers
{
    [HandleError]
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewData["Message"] = "Select files to upload";

            return View();
        }

        public ActionResult UploadResult(UploadStatus status)
        {
            return View(status);
        }
    }
}

File: src/Chapter05/SlickUpload-5.5.1/AdvancedSamples/AspNetMvcVB/Controllers/HomeController.vb
Imports Krystalware.SlickUpload.Controls
Imports Krystalware.SlickUpload.Status

<HandleError()> _
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Message") = "Select files to upload"

        Return View()
    End Function

    Function UploadResult(ByVal status As UploadStatus) As ActionResult
        Return View(status)
    End Function
End Class
